operator/controller/src/main/resources/k8s/default/app.podDisruptionBudget.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata: { }
+spec:
+  maxUnavailable: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: apicurio-registry
+      app.kubernetes.io/component: app
```

Comment:
We might have to use the `app.kubernetes.io/instance` label here as well. If there are multiple Registry instances, this PDB would apply to all of them (and vice versa). We should deploy multiple instances when testing dependent resources to check for these cases - at least in this test and in the smoke tests.

Reply:
The `labelSelector` in `@KubernetesDependent` should handle this without the label, but this is used by the Kubernetes itself.